### DISCUSSION TITLE
tui-checkbox color

### DISCUSSION THREAD

**Discussion by expdts** [taiga-family/taiga-ui#6904] [Q&A] [answered]
Подскажите пожалуйста, как я могу задать свою толщину и цвет для tui-checkbox? Сейчас его границы выглядят тускло и пользователи его с трудом видят ![image](https://github.com/taiga-family/taiga-ui/assets/109337333/ad0b233b-74aa-4df9-9491-918227244cba)

**Comment by waterplea** [comment, marked as answer]
У experimental чекбокса, который придёт на смену обычному в 4.0 все стили определены на самом хосте, так что их можно переопределять по своему усмотрению в глобальных стилях:
```css
[tuiCheckbox]:not(:checked) {
  box-shadow: 0 0 0 2px grey;
}
```
Либо можно полностью свой appearance создать и передать через DI:
https://github.com/taiga-family/taiga-ui/blob/main/projects/experimental/components/checkbox/checkbox.options.ts#L16

Вот пример создания whiteblock апиранса, который используется по умолчанию:
https://github.com/taiga-family/taiga-ui/blob/main/projects/core/styles/theme/appearance/outline.less
